src/NodaTime/Duration.cs

```diff
@@ -295,26 +299,62 @@ public long Ticks
         /// <summary>
         /// Gets the total number of minutes in this duration, as a <see cref="Double"/>.
         /// </summary>
-        /// <remarks>This property is the <c>Duration</c> equivalent of <see cref="TimeSpan.TotalMinutes"/>.</remarks>
+        /// <remarks>This property is the <c>Duration</c> equivalent of <see cref="TimeSpan.TotalMinutes"/>.
         /// Unlike <see cref="Minutes"/>, it represents the complete duration in minutes rather than
         /// the whole number of minutes within the hour. So for a duration
         /// of 2 hours, 30 minutes and 45 seconds, the <c>Minutes</c> property will return 30, but <c>TotalMinutes</c>
         /// will return 150.75.
+        /// </remarks>
         /// <value>The total number of minutes in this duration.</value>
         public double TotalMinutes => days * (double) MinutesPerDay + nanoOfDay / (double) NanosecondsPerMinute;
 
         /// <summary>
         /// Gets the total number of seconds in this duration, as a <see cref="Double"/>.
         /// </summary>
-        /// <remarks>This property is the <c>Duration</c> equivalent of <see cref="TimeSpan.TotalSeconds"/>.</remarks>
+        /// <remarks>
+        /// This property is the <c>Duration</c> equivalent of <see cref="TimeSpan.TotalSeconds"/>.
         /// Unlike <see cref="Seconds"/>, it represents the complete duration in seconds rather than
         /// the whole number of seconds within the minute. So for a duration
         /// of 10 minutes, 20 seconds and 250 milliseconds, the <c>Seconds</c> property will return 20, but <c>TotalSeconds</c>
         /// will return 620.25.
-        /// <value>The total number of minutes in this duration.</value>
+        /// </remarks>
+        /// <value>The total number of seconds in this duration.</value>
         public double TotalSeconds => days * (double) SecondsPerDay + nanoOfDay / (double) NanosecondsPerSecond;
 
-        // TODO: TotalMilliseconds, TotalTicks? What about TotalNanoseconds instead of ToDoubleNanoseconds?
+        /// <summary>
+        /// Gets the total number of milliseconds in this duration, as a <see cref="Double"/>.
+        /// </summary>
+        /// <remarks>This property is the <c>Duration</c> equivalent of <see cref="TimeSpan.TotalMilliseconds"/>.</remarks>
+        /// Unlike <see cref="Seconds"/>, it represents the complete duration in seconds rather than
```

Comment:
I think this para (and `<value/>` below) needs updating to talk about milliseconds.